src/poetry_monorepo_dependency_plugin/plugin.py

```diff
@@ -75,10 +77,26 @@ def handle(self) -> int:
         return super().handle()
 
 
+class ExportWithoutPathDepsCommand(ExportCommand):
+    name = "export-without-path-deps"
+    description = (
+        "Extends the 'export' command to generate exports in which path dependencies to "
+        "other Poetry projects are removed from package dependencies."
+    )
+
+    def handle(self) -> int:
+        path_dependency_remover = PathDependencyRemover()
+        path_dependency_remover.update_dependency_group(
+            self.io, self.poetry.pyproject, self.poetry.package.dependency_group("main")
```

Comment:
S (Substantive): this is probably fine for this version, but we likely would need to be able to handle when export is called with other groups with the `--with <groups>` option too?

Reply:
Curious how the build/publish rewrite was working with the monorepo group, given it uses a very similar implementation.